shared/protocol/src/main/java/io/pravega/shared/segment/StreamSegmentNameUtils.java

```diff
@@ -71,6 +74,12 @@
      */
     private static final String TABLES = "_tables";
 
+    /**
+     * Prefix for identifying system created mark segments for storing watermarks. 
+     */
+    @Getter(AccessLevel.PACKAGE)
+    private static final String MARK_PREFIX = "_MARK";
```

Comment:
This looks like in belongs in `NameUtils`. Should we merge these two utility classes?